data-generator/src/main/scala/com/xebialabs/xlrelease/generator/ReleasesGenerator.scala

```diff
@@ -85,6 +85,10 @@ class ReleasesGenerator {
     List(directory) ++ entries
   }
 
+  def generateFolders(amount: Int): Seq[Ci] = {
+    for (i <- 1 to amount) yield Folder.build(i)
```

Comment:
Minor thing: I think it's better if you define the folder name here, not inside the `Folder.build`, similarly to the way it's done for Releases.